scrapy/signalmanager.py

```diff
@@ -1,6 +1,7 @@
-from pydispatch import dispatcher
 from scrapy.utils import signal as _signal
 
+from pydispatch import dispatcher
+
```

Comment:
:lipstick: Third parties before the current library (Scrapy).

Reply:
I don't even know how this change got here! :shrug: